cltk/corpus/hindi/alphabet.py

```diff
@@ -31,3 +31,6 @@
 # Anusvara is used for final velar nasal sound, Visarga adds voiceless breath after vowel and Candrabindu is used to nasalize vowels 
 
 MODIFIERS = ['◌্','◌ঁ','◌ং','◌ঃ']
+
+#Om is part of the iconography found in ancient and medieval era manuscripts, temples, monasteries and spiritual retreats in Hinduism, Buddhism, and Jainism.The symbol has a spiritual meaning in all Indian dharmas, but the meaning and connotations of Om vary between the diverse schools within and across the various traditions.
```

Comment:
Please, could you add this details  in cltk docs ?

Reply:
@LBenzahia Do I need to add the details about only **Om** in cltk docs or whole [Devnagri](https://en.wikipedia.org/wiki/Devanagari) script? I also want to know should I put the details in [hindi.rst](https://github.com/cltk/cltk/blob/master/docs/hindi.rst) or somewhere else.

Reply:
@LBenzahia  I have added these details in cltk doc. Please have a look.

Reply:
@inishchith I have solved the issue related to **case** and updated the cltk docs. Please review it.

Reply:
@avkumar19 ✨Looks good